src/main/java/pl/debuguj/parkingspacessystem/exceptions/CarRegisteredInSystemException.java

```diff
@@ -0,0 +1,17 @@
+package pl.debuguj.parkingspacessystem.exceptions;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+/**
+ * Created by grzesiek on 17.10.17.
+ */
+@ResponseStatus(HttpStatus.BAD_REQUEST)
+public class CarRegisteredInSystemException extends Exception {
+
+    private static final long serialVersionUID = 1L;
```

Comment:
Co Ci daje tutaj (i w innych miejscach) `SVUID`?

Reply:
Właściwie to z ze zwykłego pośpiechu zostało, jak kilka innych rzeczy.